source/quick-start.txt

```diff
@@ -28,74 +28,99 @@ instance.
+      First, install Git using `Git's Getting Started Guide
+      <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__.
 
-.. code-block:: none
+      Then, in your shell, clone the `js-starter repository
+      <https://github.com/mongodb-university/js-starter>`__:
 
-   mkdir node_quickstart
+      .. code-block:: bash
 
-Then, navigate into that directory so you can work directly with your
-project's files:
+         git clone https://github.com/mongodb-university/js-starter.git
 
-.. code-block:: none
+      Next, navigate into the repository:
 
-   cd node_quickstart
+      .. code-block:: bash
 
-Next, set up NPM for your project by running the following command.
-This command creates a file called ``package.json``:
+         cd js-starter
 
-.. code-block:: none
+      Then, install the Node.js driver:
 
-   npm init -y
+      .. code-block:: none
 
-.. note:: Why the -y?
+         npm install
 
-   If you specify the ``-y`` option in the command, NPM automatically
-   accepts the default values for the command. Omit the ``-y`` flag to
-   interactively select your project settings.
+      This command performs the following actions:
 
-Add MongoDB as a Dependency
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      - Downloads and saves the ``mongodb`` package in a directory
+        called ``node_modules``
+      - Installs ``eslint``, which the project uses to identify and
+        report issues in your code
+      - Installs ``prettier``, which the project uses to format your code uniformly
 
-Next, add the official Node.js MongoDB driver to your project dependencies.
-Use the following command to instruct NPM to download and install the
-``mongodb`` package.
+   .. tab:: Create the Project
+      :tabid: create the project
 
-.. code-block:: none
+      First, in your shell, create a directory for your project:
+
+      .. code-block:: bash
+
+         mkdir node_quickstart
+
+      Then, navigate into that directory:
+
+      .. code-block:: bash
+
+         cd node_quickstart
+
+      Next, initialize your project:
+
+      .. code-block:: bash
+
+         npm init -y
+
+      .. tip:: Why the -y?
+
+         If you specify the ``-y`` option in the command, npm uses the
+         default values for your project settings. If you want to
+         interactively select your project settings, omit the ``-y`` flag.
+
+      Then, install the Node.js driver:
+
+      .. code-block:: none
+
+         npm install mongodb
 
-   npm install mongodb
+      This command performs the following actions:
 
-This command downloads ``mongodb`` package and dependencies required for its
-installation and saves them into a directory called ``node_modules`` in
-your project directory, and records the dependency information in the
-``package.json`` file you generated in the previous step.
+      - Downloads the ``mongodb`` package and dependencies required
+      - Saves the package in a directory called ``node_modules``
```

Comment:
Nit: I think this can have more parallel structure to the following list item

```suggestion
      - Saves the package in the``node_modules`` directory
```